lib/improver/nowcasting/utilities.py

```diff
@@ -42,6 +42,63 @@
     extract_nearest_time_point, iris_time_to_datetime)
 
 
+class ExtendRadarMask(object):
+    """
+    Extend the mask on radar rainrate data based on the radar coverage
```

Comment:
Again, I think we should be talking about precip from radar, or am I barking up the wrong tree?

Reply:
I think you're barking up the wrong tree but we could ask Stephen for a second opinion?